libmscore/edit.cpp

```diff
@@ -2499,6 +2499,15 @@ void Score::cmdDeleteSelection()
                               tick = toRest(e)->tick();
                         else if (e->isSpannerSegment())
                               tick = toSpannerSegment(e)->spanner()->tick();
+                        else if (e->isBreath()) {
+                              // we want the tick of the ChordRest that precedes the breath mark (in the same track)
+                              for (Segment* s = toBreath(e)->segment()->prev(); s; s = s->prev()) {
+                                    if (s->isChordRestType() && s->element(e->track())) {
```

Comment:
I'd prefer to also see nullptr check here

Reply:
A nullptr check for what? s? That is handled by the loop condition.